api/src/main/java/org/openmrs/Provider.java

```diff
@@ -17,20 +40,42 @@
  *
  * @since 1.9
  */
+@Entity
+@Table(name = "provider")
+@SecondaryTable(name = "provider_unused_fields")
+@AttributeOverrides({ @AttributeOverride(name = "name", column = @Column),
+        @AttributeOverride(name = "description", column = @Column(table = "provider_unused_fields", name = "description", insertable = false, updatable = false)) })
```

Comment:
Do we wanna add a comment explaining why we introduced `provider_unused_fields`?

Reply:
Done

Reply:
What would be the effect of using the same `unused_fields` table like we already have `users_unused_fields` instead of creating a new one for each of these scenarios?

Reply:
Should be able to work